### PR TITLE
Feature/guest

### PR DESCRIPTION
**Pull request by mloeffle**

### CODE REVIEW

File: components/account/AccountRegister.vue [hunk 1]
```diff
@@ -41,6 +63,11 @@ const salutationOptions = computed(
 );
 
 const currentCountry = computed(() => sessionContext.countryId.value);
+const passwordRequired = ref(true);
+const handleGuestChange = fields => {
+    console.log(fields.target.checked);
```
Comment (jkniest) [inline components/account/AccountRegister.vue:68]:
```suggestion
```

File: pages/search/index.vue [hunk 2]
```diff
@@ -32,7 +32,8 @@ setInitialListing(productSearch.value as Schemas['ProductListingResult']);
     <div class="container">
         <h1 class="mb-6 text-center">
             <span v-if="products?.length">
-                results for "<strong>{{ searchTerm }}</strong>"
+                results for "<strong>{{ searchTerm }}</strong
```
Comment (jkniest) [inline pages/search/index.vue:35]: Dont know what happend here but please revert

**Review by jkniest — CHANGES_REQUESTED** [review]

File: pages/search/index.vue (hunk 2 above)
Comment (mloeffle) [reply pages/search/index.vue:35]: the linting is responsible for this actually. something seems to get confused by the tag between double quotes

File: components/Utility/UtilityPill.vue [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
     },
 );
 </script>
 
 <template>
     <span
-        v-if="number > 0"
-        class="grid h-3.5 min-w-3.5 items-center justify-center rounded-full bg-brand-primary p-0.5 text-xs leading-none text-white outline outline-1 outline-white"
+        v-if="content"
+        class="h-3.5 min-w-3.5 rounded-full bg-brand-primary p-0.5 text-xs leading-none text-white outline outline-1 outline-white"
```
Comment (aheartforspinach) [inline components/Utility/UtilityPill.vue:15]: why did we remove `grid items-center justify-center`?

File: components/account/AccountRegister.vue [hunk 4]
```diff
@@ -22,8 +38,14 @@ const handleRegisterSubmit = async (fields: FormkitFields) => {
             ...fields,
         });
         isLoading.value = false;
-        navigateTo('/account');
+
+        if (props.redirectAfterSuccess) {
+            navigateTo(props.redirectTarget);
+        }
+
+        pushSuccess('You successfully logged in as a guest.');
```
Comment (aheartforspinach) [inline components/account/AccountRegister.vue:46]: I'm confused, sorry :D Why are we logged in as a guest?

File: components/Utility/UtilityPill.vue (hunk 3 above)
Comment (mloeffle) [reply components/Utility/UtilityPill.vue:15]: just checked again and added flex instead. but the reason in general was, that `grid` didn't have any impact on how the pill wad styled. which positions the content vertically more centered while grid did not as far as i could see on my devices

File: components/account/AccountRegister.vue (hunk 4 above)
Comment (mloeffle) [reply components/account/AccountRegister.vue:46]: :D wasn't sure about this too but wanted to keep the consistency and have some kind of feedback, but maybe its better to remove this and let the change of the form be enough

File: components/Account/AccountRegister.vue [hunk 5]
```diff
@@ -167,6 +193,21 @@ const currentCountry = computed(() => sessionContext.countryId.value);
             <span>your account data</span>
         </div>
 
+        <FormKit
+            v-if="allowGuest"
+            type="checkbox"
+            label="dont create a customer account"
```
Comment (m-borgmann) [inline components/Account/AccountRegister.vue:199]: Translation

File: components/Account/AccountRegister.vue [hunk 6]
```diff
@@ -21,8 +37,14 @@ const handleRegisterSubmit = async (fields: FormkitFields) => {
             ...fields,
         });
         isLoading.value = false;
-        navigateTo('/account');
+
+        if (props.redirectAfterSuccess) {
+            navigateTo(props.redirectTarget);
+        }
+
+        pushSuccess('You successfully logged in');
```
Comment (m-borgmann) [inline components/Account/AccountRegister.vue:45]: Translations hier und in 47

**Review by m-borgmann — APPROVED** [review]

**Comment by mloeffle** [comment]
On Hold until config route is implemented to toggle the functionality.

File: i18n/de-DE/account.json [hunk 7]
```diff
@@ -133,7 +133,13 @@
             "terms": {
                 "label": "AGB",
                 "help": "Akzeptieren Sie unsere Allgemeinen Geschäftsbedingungen?"
-            }
+            },
+            "guest": {
+                "label": "Gast",
+                "toggle": "Ich möchte keinen Account erstellen"
+            },
```
Comment (m-borgmann) [inline i18n/de-DE/account.json:140]: Brauchen wir hierfür keine englischen Translations?

**Review by aheartforspinach — APPROVED** [review]

**Pull request merged**